src/redux/details/apiCall.js

```diff
@@ -1,17 +1,17 @@
 // import { useParams } from "react-router-dom";
-import { getDetailsSuccess } from './detailsSlice';
+// import { getDetailsSuccess } from './detailsSlice';
 
-const fetchDetails = (roomId) => async (dispatch) => {
-  const URL = `https://dummyjson.com/products/${roomId}`;
+// const fetchDetails = (roomId) => async (dispatch) => {
+//   const URL = `https://dummyjson.com/products/${roomId}`;
 
-  await fetch(URL)
-    .then((response) => response.json())
-    .then((data) => {
-      dispatch(getDetailsSuccess(data));
-    })
-    .catch((error) => {
-      console.log(error);
-    });
-};
+//   await fetch(URL)
+//     .then((response) => response.json())
+//     .then((data) => {
+//       dispatch(getDetailsSuccess(data));
+//     })
+//     .catch((error) => {
+//       console.log(error);
+//     });
+// };
 
-export default fetchDetails;
+// export default fetchDetails;
```

Comment:
- [ ] Please avoid pushing commented code to the project. If you are worried about losing the code, we can always look at the previous version using GIT

Reply:
Hi @williamrolando88, that's so weird this file is there because I have actually deleted it. Since all the API fetch logic is now in the slice I didn't need this file anymore. I thought I might have not pushed after deleting the file but when I try pushing it says it's up to date....